### ISSUE TITLE
[Windows] Correct syntax for remote git repo?

### ISSUE THREAD

**Issue by Borromini** [IJHack/QtPass#510]
Hi!

I've been trying to get QtPass to work on Windows 10 with my remote (local file server) git repo (shared setup between my Linux clients (CLI pass), smartphone (Android Password Store) and a Windows desktop).

I am trying to get QtPass to talk to my git repo but it's not working out. I've installed a basic GnuPG for Windows (key has been recognised and imported), and the Windows binary for Git as well. I've seen people suggest using git--gui-askpass (which Windows does not seem to see as a binary?) instead of the plain Windows git binary, but it is not getting me any further. I'm not seeing any logging paths in the configuration settings.

**Binary paths**
![Pass binary paths](https://user-images.githubusercontent.com/8218431/74606354-19883d00-50d0-11ea-8885-937bf8e78857.png)

**SSH Path**
![Pass SSH path](https://user-images.githubusercontent.com/8218431/74606358-24db6880-50d0-11ea-91fe-f05b2df68dd1.png)

Any help would be appreciated.

**Comment by hannes08** [comment]
Hi Borromini! 

tl;dr: You need to clone the git repository locally first, then add it's path as a profile to QtPass.

For example, do a "git clone user@10.0.0.20:/home/user/git/pass-store.git" in your local Documents folder. Then add a profile in QtPass which points to C:\Users\<user>\Documents\<git repo folder>

As a sidenote: I personally found it easier to user https based git synchroization on windows. It will store the password in the windows credential manager.

**Comment by annejan** [comment]
Git remote syntax question - not a QtPass bug. Please use proper git syntax for remotes.
